gen/generator.go

```diff
@@ -764,16 +762,30 @@ func getExampleCode(filePath string) []byte {
 	return content
 }
 
-// When GEN_CLASSES environment variable is set, the class metadata is retrieved from the APIC or devnet docs and stored in the meta directory.
-func getClassMetadata() {
+// When RE_GEN_CLASSES environment variable is set, the existing class metadata is retrieved from APIC or the latest devnet docs and stored in the meta directory.
+func reGenClassMetadata() {
+	_, re_gen_classes := os.LookupEnv("RE_GEN_CLASSES")
```

Comment:
Does LookupEnv function make distinction between true("1") / false ("0")? From my understanding of the function it does not, and only check if the env var is set.

// LookupEnv retrieves the value of the environment variable named
// by the key. If the variable is present in the environment the
// value (which may be empty) is returned and the boolean is true.
// Otherwise the returned value will be empty and the boolean will
// be false.

So I think in the case if RE_GEN_CLASSES: '0' you would still trigger a re-generate, which I assume you do not want?

Reply:
I think you are correct. I have adjusted RE_GEN_CLASSES and GEN_ANNOTATION_UNSUPPORTED to only accept strings representing true boolean values. Anything else is considered false.